SI/Sem.02/Pract.02/Task7.cpp

```diff
@@ -0,0 +1,27 @@
+
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    char symbol;
+    cout << "Enter a symbol: ";
+    cin >> symbol;
+    if (symbol >= 65 && symbol <= 90)
```

Comment:
The program works correctly. However, if you have to search each symbol's code every time, it takes time. When you have an exam, you will not be allowed to use the Internet to check the ASCII table and you do not need to learn it by heart. My advice is to use the symbols themselves:
if (symbol >= 'A' && symbol <= 'Z')
symbol -= 'a'-'A';